gym/wrappers/record_episode_statistics.py

```diff
@@ -56,35 +159,35 @@ def reset(self, **kwargs):
 
     def step(self, action):
         """Steps through the environment, recording the episode statistics."""
+        infos_processor = self.stats_info_processor(self.num_envs)
```

Comment:
Do we need to initialise this with every step?

Reply:
So, I'm initializing now in the `__init__` and in the `step` just resetting `info_processor.info` to an empty dict. Seems better?

Reply:
Yes, amazing, thanks